Tests/MongoSwiftTests/ChangeStreamTests.swift

```diff
@@ -3,8 +3,98 @@ import mongoc
 import Nimble
 import XCTest
 
-final class ChangeStreamTest: MongoSwiftTestCase {
-    func testChangeStream() throws {
+final class ChangeStreamTests: MongoSwiftTestCase {
+    func testChangeStreamOnAClient() throws {
+        guard MongoSwiftTestCase.topologyType != .single else {
+            print("Skipping test case because of unsupported topology type \(MongoSwiftTestCase.topologyType)")
+            return
+        }
+
+        let client = try MongoClient()
+        guard try client.serverVersion() >= ServerVersion(major: 4, minor: 0) else {
+            print("Skipping test case for server version \(try client.serverVersion())")
+            return
+        }
+
+        let changeStream = try client.watch()
+        let db1 = client.db("db1")
+        defer { try? db1.drop() }
+        let coll1 = db1.collection("coll1")
+        let coll2 = db1.collection("coll2")
+        try coll1.insertOne(["a": 1])
+        try coll2.insertOne(["x": 123])
+
+        let res1 = changeStream.next()
+        // expect the change stream to contain a change document for the `insert` operation
+        expect(res1).toNot(beNil())
+        expect(res1?.operationType).to(equal(.insert))
+        expect(res1?.fullDocument?["a"]).to(bsonEqual(1))
```

Comment:
are you just comparing the one field "a" because of `_id`?
if you want, you can choose `_id` yourself and then it will always be predictable. for ex.:
```swift
let doc1: Document = ["a": 1, "_id": 1]
try coll1.insertOne(doc1)
...
expect(res1?.fullDocument).to(equal(doc1))
```